packages/components/src/theme/ThemeModel.ts

```diff
@@ -43,6 +43,7 @@ export type ThemeIconsRequiringManualColorChanges =
 
 export const DEFAULT_DARK_THEME_KEY = 'default-dark' satisfies BaseThemeKey;
 export const DEFAULT_LIGHT_THEME_KEY = 'default-light' satisfies BaseThemeKey;
+export const THEME_KEY_OVERRIDE_QUERY_PARAM = 'themeKey';
```

Comment:
Should we bother with the `Key` part of this param? May as well just call it `theme`.

Reply:
Didn't see a response to this - why not just call it `theme`?

Reply:
ah, didn't see this comment. I have renamed it.